CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ find_package(OMECompat 5.2.0 REQUIRED)
 find_package(OMECommon 5.2.0 REQUIRED)
 find_package(OMEXML 5.2.0 REQUIRED)
 find_package(OMEFiles 0.2.3 REQUIRED)
-find_package(BioFormatsJACE 5.2.1)
+find_package(BioFormatsJACE 5.3.3)
```

Comment:
Yes.  Please see https://github.com/openmicroscopy/ome-files-performance/pull/19 for the replacement.